src/main/resources/tinylog.properties

```diff
@@ -11,3 +11,14 @@ exception = strip: jdk.internal
 #level@org.jabref.gui.maintable.PersistenceVisualStateTable = debug
 
 level@org.jabref.http.server.Server = debug
+
+# FIXME: Remove before merging the branch
```

Comment:
We should not forget about this.

I also know that having this in helps to quickly debug. Maybe, we should leave it as is (and replace the "FIXME" by "AI")